package.json

```diff
@@ -82,4 +82,4 @@
         "typescript": "^3.6.4",
         "watch": "^1.0.2"
     }
-}
+}
```

Comment:
Check your vscode settings, it seems it routinely removes this EOF for package.json